Sources/TensorFlow/Operators/NN.swift

```diff
@@ -434,6 +529,31 @@ public func conv3D<Scalar: TensorFlowFloatingPoint>(
         padding: padding.raw)
 }
 
+/// Computes a 2-D Depthwise convolution using `self` as input, with the specified
+/// filter, strides, and padding.
+///
+/// - Parameters:
+///   - x: The input.
+///   - depthwise_kernel: The depthwise convolution filter.
+///   - strides: The strides of the sliding filter for each dimension of the input.
+///   - padding: The padding for the operation.
+/// - Precondition: `self` must have rank 4.
```

Comment:
We usually wrap numeric values in backquotes _except_ when they are not used to describe values that can manifest in variables. In this case `4` is describing the value of `self`, and thus should be backquoted.
```suggestion
/// - Precondition: `self` must have rank `4`.
```

Reply:
@rxwei none of the `self` preconditions have this enveloped in backquotes.

Reply:
But if `self` doesn't anymore shouldn't we just remove this comment and say the input must have rank `4`?

Reply:
`` `self` `` should be changed to `` `input` ``. `4` should be changed to `` `4` ``.